compiler/noirc_frontend/src/ownership/mod.rs

```diff
@@ -320,6 +320,19 @@ impl Context {
     }
 
     fn handle_call(&mut self, call: &mut crate::monomorphization::ast::Call) {
+        // Hack: We don't have references yet but pretend the array length builtin
```

Comment:
```suggestion
        // Hack: We don't have references of `self` yet but pretend the array length builtin
```
A bit pedantic but would this not be more accurate?

Also do we have an issue for references to self we want to link here?

Reply:
That'd be less accurate - I mean we don't have immutable references in general currently. We do have `&mut self` which could be used but we don't want array len to take a mutable self reference. So it takes a `self` but we don't actually want to inc_rc just for a .len() call, hence this hack.